acl/src/test/java/org/apache/rocketmq/acl/plain/PlainPermissionManagerTest.java

```diff
@@ -247,7 +259,7 @@ public void multiFilePathTest() {
 
     @Test
     public void testWatch() throws IOException, IllegalAccessException, InterruptedException {
-        File file = new File("src/test/resources");
+        File file = new File(PATH);
         System.setProperty("rocketmq.home.dir", file.getAbsolutePath());
 
         String fileName = System.getProperty("rocketmq.home.dir") + File.separator + "/conf/acl/plain_acl_test.yml";
```

Comment:
This line will not be compatible with Windows

Reply:
Thank you for your advice

I have changed to it as follows

`private static final String PATH = PlainPermissionManagerTest.class.getResource(File.separator).getFile();`